controllers/cinderapi_controller.go

```diff
@@ -721,6 +722,11 @@ func (r *CinderAPIReconciler) reconcileNormal(ctx context.Context, instance *cin
 			err.Error()))
 		return ctrlResult, err
 	} else if (ctrlResult != ctrl.Result{}) {
+		instance.Status.Conditions.MarkFalse(
```

Comment:
As per [1], I'm wondering if this should be marked to `Unknown` instead of `False`, because we're comparing `ctrlResult` with `ctrl.Result{}`, and not returning an `err` like the previous if on L716 (so it's like this is in-progress, and not blocked by an error).

[1] https://github.com/openstack-k8s-operators/docs/blob/main/conditions.md#conditions-in-openstack-k8s-operators-podified-controlplane-operators

Reply:
I think this should be consistent with how this is handled in similar situations, such as the one at L701. And even if there's no error, it's still accurate to say TLSInputReadyCondition is not ready.

Reply:
I think that L701 should also be marked as `Unknown` instead of `False` because it's not something set when `err != nil`, but something caught by a change in the `ctrlResult` when the Certificate is Validate. IIUC that seems more an in-progress operations that will return and be fixed by the next Reconcile loop.
However, I think this is really a minor thing compared to the goal of this patch, so if you think that `False` fits better the meaning of this particular situation, I'm ok with that.